### DISCUSSION TITLE
Blockcraftery fixes

### DISCUSSION THREAD

**Discussion by xJon** [ACGaming/UniversalTweaks#157] [Bugfix Suggestions]
https://github.com/MysticMods/Blockcraftery/issues/49

"Not urgent, would be nice!"

**Comment by xJon** [comment]
Fixing https://github.com/MysticMods/Blockcraftery/issues/85 would also be a nice bonus.

**Comment by WaitingIdly** [comment]
[Blockcraftery: Refurbished](https://www.curseforge.com/minecraft/mc-mods/blockcraftery-refurbished) recently released, which fixes both of your issues

**Reply by xJon** [reply]
Thanks for letting me know! It requires Cleanroom Loader which makes it a bit less convenient but will still keep in mind.
